### ISSUE TITLE
Actor properties window.

### ISSUE THREAD

**Issue by Snuggx** [OldUnreal/UnrealTournamentPatches#105]
It's a bit annoying that the Actor Properties window since 469a disappears when you click on the grid. It shouldn't go away, or at least be switchable.

**Comment by CacoFFF** [comment]
I agree

**Comment by an-eternity** [comment]
A little issue that matches the title "Actor Properties Window", but not with UnrealEd...
Sometimes, if client is in windowed mode and 2 or more Actor Properties windows are opened, after several switching between main client window and these windows (by using mouse left button to click in any arbitrary area inside of these windows) the crash happens:

`Critical: FPropertyItem::SendToControl`
`Critical: FPropertyItem::OnKillFocus`
`Critical: WProperties::SetItemFocus`
`Critical: WProperties::OnActivate`
`Critical: WM_ACTIVATE`
`Critical: MessagePump`
`Critical: MainLoop`
`Exit: Executing UObject::StaticShutdownAfterError`
`Exit: Executing UWindowsClient::ShutdownAfterError`
`DevMusic: Unregister music: Music Suprfist.Suprfist`
`DevMusic: glxStopMusic music: Music Suprfist.Suprfist`
`Exit: UGalaxyAudioSubsystem::ShutdownAfterError`
`Log: DirectDraw End Mode`
`Exit: UD3D9RenderDevice::ShutdownAfterError`
`Exit: Exiting.`

Same way this happens with XOpenGL renderer, and with any older client version (v436-v451) for Windows.
Maybe this problem is with OS (Windows 7 Pro. 64-bit) or with graphics driver...

**Comment by stijn-volckaert** [comment]
Indeed. I fixed this on November 3. Will upload a new snapshot soon

**Comment by an-eternity** [comment]
@SeriousBuggie, Please, create that issue if you can reproduce this crash in your environment. I am not yet sure whether it is problem on my side, with OS, drivers, or some settings...

**Comment by SeriousBuggie** [comment]
Fixed in 469b 10 Nov. Confirmed.

But it make in odd way. If select nothing then Actor Properties still work with previous selection. It is not good because mess you "I am deselect or not". In v436 it work more properly - empty window with title "nothing selected".

**Comment by Snuggx** [comment]
> 
> 
> Fixed in 469b 10 Nov. Confirmed.
> 
> But it make in odd way. If select nothing then Actor Properties still work with previous selection. It is not good because mess you "I am deselect or not". In v436 it work more properly - empty window with title "nothing selected".

Yep, this is very odd.
That why i jumped back to an august release with my editor UT.

**Comment by SeriousBuggie** [comment]
> @SeriousBuggie, Please, create that issue if you can reproduce this crash in your environment. I am not yet sure whether it is problem on my side, with OS, drivers, or some settings...

I often use `editactor` windows for debug and not see such crash. If you able stable reproduce it on latest test builds - please create new issue with exact reproduce steps.

I can make crash UnrealEd only if Left open `editactor` windows from previous map, and now load another one. And even that not in all cases.
